lib/app/key_customization/views/key_customization_dialog.dart

```diff
@@ -0,0 +1,350 @@
+/*
+ * Copyright (C) 2024 Yubico.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+import '../../../android/state.dart';
+import '../../../core/state.dart';
+import '../../../management/models.dart';
+import '../../../widgets/app_input_decoration.dart';
+import '../../../widgets/app_text_form_field.dart';
+import '../../../widgets/focus_utils.dart';
+import '../../../widgets/responsive_dialog.dart';
+import '../../models.dart';
+import '../../state.dart';
+import '../../views/device_avatar.dart';
+import '../../views/keys.dart';
+import '../models.dart';
+import '../state.dart';
+
+extension _ColorHelper on String? {
+  // ignore: unused_element
+  Color? asColor() {
+    final hexValue = this;
+    if (hexValue == null) {
+      return null;
+    }
+
+    final intValue = int.tryParse(hexValue, radix: 16);
+    if (intValue == null) {
+      return null;
+    }
+
+    return Color(intValue);
+  }
+}
```

Comment:
Why do we need this if it's unused?

Reply:
this has been moved to different file and is used now.